Example/Tests/TagsFlowLayoutTest.swift

```diff
@@ -0,0 +1,246 @@
+//
+//  TagsFlowLayoutTest.swift
+//  collection_flow_layout_Example
+//
+//  Created by sergey on 2/28/18.
+//  Copyright © 2018 CocoaPods. All rights reserved.
+//
+
+import Quick
+import Nimble
+import collection_flow_layout
+
+class TagsFlowLayoutTest: QuickSpec {
+    private let kScreenWidth: CGFloat = 414
+    private let kScreenHeight: CGFloat = 736
+    
+    override func spec() {
+        let items = ["Twitter", "Facebook", "Instagram", "A", "B", "Frameworks", "Developers", "Source code"]
+        
+        describe("Check tags flow layout") {
+            it("tags flow layout fields validation") {
```

Comment:
You need to cover with tests all if cases. You can run tests with code coverage util and check what is not covered.